Pod/Classes/TSMessageView.m

```diff
@@ -303,10 +305,11 @@ - (id)initWithTitle:(NSString *)title
         [self.titleLabel setShadowColor:[UIColor colorFromHexString:[current valueForKey:@"shadowColor"]]];
         [self.titleLabel setShadowOffset:CGSizeMake([[current valueForKey:@"shadowOffsetX"] floatValue],
                                                     [[current valueForKey:@"shadowOffsetY"] floatValue])];
+        self.titleLabel.minimumScaleFactor = 0.75;
+        self.titleLabel.adjustsFontSizeToFitWidth = YES;
         
-        int64_t titleNumberOfLines = [[current valueForKey:@"titleNumberOfLines"] intValue];
-        self.titleLabel.numberOfLines = titleNumberOfLines;
-        self.titleLabel.lineBreakMode = [[current valueForKey:@"lineBreakMode"] intValue];
+        self.titleLabel.numberOfLines = 1;
+        self.titleLabel.lineBreakMode = NSLineBreakByTruncatingTail;
```

Comment:
I remember our entry notif titles (especially in the two name case) would occasionally wrap arround, have we tested long titles just to make sure the behavior is what we expect/want?